e2e/utils/visual-regression.js

```diff
@@ -3,9 +3,30 @@ function getMisMatchPercentage(results) {
 }
 
 module.exports.takeScreenshotAndGetWholePageCompareResult = (misMatchTolerance) => {
-  return ((getMisMatchPercentage(browser.checkViewport()) <= misMatchTolerance) || false);
+  const misMatchPercentage = getMisMatchPercentage(browser.checkViewport());
+  const isTestPassed = (misMatchPercentage <= misMatchTolerance) || false;
+  if (isTestPassed) {
+    return isTestPassed;
+  }
+  console.log('misMatchTolerance: ' + misMatchTolerance
```

Comment:
this is really verbose imho. usually we dont need this. my favourite tool for logging verbose messages for myself but not for others is [`debug`](https://www.npmjs.com/package/debug).

This is one the standard ways of logging this in node community

Reply:
Thanks for the advice,what about I add the debug in an other PR?

Reply:
sure!